packages/react-components/react-combobox/src/components/Combobox/useCombobox.tsx

```diff
@@ -184,6 +185,13 @@ export const useCombobox_unstable = (props: ComboboxProps, ref: React.Ref<HTMLIn
         })
       : undefined;
 
+  const portalSlot: Slot<typeof Portal> | undefined =
+    open || hasFocus
+      ? resolveShorthand(props.portal, {
+          required: true,
+        })
+      : undefined;
```

Comment:
Unlike listbox, portal is always rendered, so would it make sense to remove the conditional?